### ISSUE TITLE
Remove `--disable-instruction-limit` from the Playground mode

### ISSUE THREAD

**Issue by reeelix** [LMH01/alpha_tui#67]
In Playground mode, only one instruction is executed at a time. Therefore, applying a limit to the number of executed instructions doesn't make sense there.  
  
Since the option is hidden in the CLI help and will rarely be necessary, this doesn't make to much of an impact and can be considered very non-urgent.

**Comment by reeelix** [comment]
not urgent

**Comment by LMH01** [comment]
Yeah, this option could be removed from the sandbox mode. But as this is not urgent, because it is hidden by default and because the implementation could be a bit of a pain (probably requiring a new argument group) this issue will not be actively worked on for now.

**Comment by LMH01** [comment]
Idea how this could be done. As there is now a new argument group (CheckLoadArgs) this '--disable-instruction-limit' argument could be moved there. This way the argument would no longer applicable to the playground subcommand.
